crates/router/src/connector/adyen/transformers.rs

```diff
@@ -756,6 +763,9 @@ pub struct MomoData {}
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct TouchNGoData {}
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct SwishData {}
```

Comment:
This struct is not necessary as we are already using `[serde(tag = "type")]` for AdyenPaymentMethod

Reply:
removed struct ``SwishData``